scripts/newinstall.sh

```diff
@@ -217,6 +214,43 @@ if true; then
 	fi
 fi
 
+##########	Test/warn about the existence of ~/.eups
+
+if true; then
+	if [[ -e ~/.eups ]]; then
+		TIMESTAMP=$(date +%Y%m%d%S)
+		cat <<-EOF
+
+		Problems have been observed when re-installing or upgrading EUPS while
+		preserving a pre-existing ~/.eups directory.  This directory primarily
+		contains cache data and is safe to remove when EUPS is not actively
```

Comment:
This may cause not-very-advanced users to lose any customizations they have made such as declarations of personal tags, so "safe" is inappropriate.  I'm rather uncomfortable with this approach to solving the problem in the general case; I'd rather find the root cause first (but there is no rush on doing so as no one else has run into it).  Also "preforming" -> "performing".